### PR TITLE
samples: sensor: added general samples for triggers

### PR DESCRIPTION
**Pull request by CZKikin**
We wanted to introduce a fxls8974 sample, but it was declined, so we do not add another driver specific example. We wanted to have a sample that showcases the trigger functionality on some sensors and at the same time have a way to test if it works.

### CODE REVIEW

File: samples/sensor/accel_trig/README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.. _accel_trig:
```
Comment (MaureenHelm) [inline samples/sensor/accel_trig/README.rst:1]: Add `zephyr:code-sample:` directive

File: samples/sensor/magn_trig/README.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.. _magn-trig-sample:
```
Comment (MaureenHelm) [inline samples/sensor/magn_trig/README.rst:1]: Add `zephyr:code-sample:` directive

File: samples/sensor/magn_trig/README.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+Basig Magnetometer Sample With Triggers
+#######################################
+
+Overview
+********
+
+Sample application that reads magnetometer (X, Y, Z) data from
+the available device that implements SENSOR_TRIG_DATA_READY and SENSOR_CHAN_MAGN_*.
```
Comment (MaureenHelm) [inline samples/sensor/magn_trig/README.rst:10]: Add build command and expected output

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
This is a nice change, thank you.

File: samples/sensor/magn_trig/README.rst (hunk 2 above)
Comment (MaureenHelm) [reply samples/sensor/magn_trig/README.rst:1]: I see this was done on the accel_trig sample, but it needs to be done here too.

**Comment by CZKikin** [comment]
@MaureenHelm Is there something else that I should fix?

File: samples/sensor/magn_trig/README.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+   lis2dh@19 (x, y, z):    (  -6.013728,   -3.064320,    7.277760)
+   lis2dh@19 (x, y, z):    (  -6.128640,   -3.026016,    7.201152)
+   lis2dh@19 (x, y, z):    (  -6.090336,   -3.064320,    7.162848)
+   lis2dh@19 (x, y, z):    (  -6.128640,   -3.026016,    7.354368)
+   lis2dh@19 (x, y, z):    (  -6.166944,   -3.102624,    7.277760)
+   lis2dh@19 (x, y, z):    (  -6.128640,   -2.987712,    7.277760)
+   lis2dh@19 (x, y, z):    (  -6.052032,   -2.987712,    7.277760)
+   lis2dh@19 (x, y, z):    (  -6.166944,   -2.987712,    7.239456)
+   lis2dh@19 (x, y, z):    (  -6.090336,   -3.026016,    7.201152)
```
Comment (MaureenHelm) [inline samples/sensor/magn_trig/README.rst:44]: This is a magnetometer sample, but lis2dh is an accelerometer

**Review by ubieda — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**